### DISCUSSION TITLE
is there a reason the storms do not pick people up

### DISCUSSION THREAD

**Discussion by Jaytv25** [CosmicHikari/Weather2-Remastered#55] [Questions] [answered]
i was in a storm in survival and for some reason it does not pick anyone up but mobs and blocks that are not related to the players

**Comment by Mrbt0907** [comment]
Could be that your grab settings in the advanced settings are not on. There is a desync issue with the ez gui possibly, so check that out to see if that will fix your issue.

**Comment by CosmicHikari** [comment, marked as answer]
Answered by #54
